examples/rex_planar_quadrotor_lqr.jl

```diff
@@ -135,54 +143,348 @@ n0, = RD.dims(model_real)
 # ylims!((-1.25,0.75))
 
 #############################################
-## Plot fit error vs regularization 
+## Plot tracking error vs regularization 
 #############################################
 
-regularizers = exp10.(-4:2)
+x0_sampler = Product([
+    Uniform(-2.0,2.0),
+    Uniform(-2.0,2.0),
+    Uniform(-deg2rad(70),deg2rad(70)),
+    Uniform(-1.0,1.0),
+    Uniform(-1.0,1.0),
+    Uniform(-0.5,0.5)
+])
+
+x0_test = [rand(x0_sampler) for i = 1:100]
+t_sim = 5.0
+
+Qlqr = Diagonal([10.0, 10.0, 10.0, 1.0, 1.0, 1.0])
+Rlqr = Diagonal([1e-4, 1e-4])
+
+regularizers = exp10.(-5:2)
 errors = map(regularizers) do reg
     model_eDMD = run_eDMD(X_train, U_train, dt, eigfuns, eigorders, reg=reg, name="planar_quadrotor_eDMD")
     model_jDMD = run_jDMD(X_train, U_train, dt, eigfuns, eigorders, dmodel_nom, reg=reg, name="planar_quadrotor_jDMD")
+    model_eDMD_projected = EDMD.ProjectedEDMDModel(model_eDMD)
+    model_jDMD_projected = EDMD.ProjectedEDMDModel(model_jDMD)
+
+    lqr_eDMD_projected = LQRController(
+    model_eDMD_projected, Qlqr, Rlqr, xe, ue, dt, max_iters=10000, verbose=true
+    )
+    lqr_jDMD_projected = LQRController(
+        model_jDMD_projected, Qlqr, Rlqr, xe, ue, dt, max_iters=10000, verbose=true
+    )
+    
+    error_eDMD_projected = mean(test_initial_conditions(dmodel_real, lqr_eDMD_projected, xe, x0_test, t_sim, dt))
+    error_jDMD_projected = mean(test_initial_conditions(dmodel_real, lqr_jDMD_projected, xe, x0_test, t_sim, dt))
+    (;error_eDMD_projected, error_jDMD_projected)
 
-    # Check test errors
-    olerr_eDMD = EDMD.open_loop_error(model_eDMD, X_test, U_test)
-    olerr_jDMD = EDMD.open_loop_error(model_jDMD, X_test, U_test)
-    fiterr_eDMD = BilinearControl.EDMD.fiterror(model_eDMD, X_test, U_test)
-    fiterr_jDMD = BilinearControl.EDMD.fiterror(model_jDMD, X_test, U_test)
-    (;olerr_eDMD, olerr_jDMD, fiterr_eDMD, fiterr_jDMD)
 end
 fields = keys(errors[1])
 res = Dict(Pair.(fields, map(x->getfield.(errors, x), fields)))
-p_ol = @pgf Axis(
+
+p_tracking = @pgf Axis(
     {
         xmajorgrids,
         ymajorgrids,
         xmode = "log",
-        xlabel = "regularization value",
-        ylabel = "open loop error",
+        ymode = "log",
```

Comment:
maybe this should be normal (non-log) scale? I think it'd help accentuate the "curve" of the EDMD plot to show that it has a single, local minima while jDMD is flat for a long time. Could also think about placing a "mini" window in the top-right corner showing the non-log-scaled plot.